python/cuml/_thirdparty/sklearn/preprocessing/_data.py

```diff
@@ -1529,8 +1575,7 @@ def transform(self, X):
 
         if sparse.isspmatrix_csr(X):
             if self.degree > 3:
-                res = self.transform(X.tocsc()).tocsr()
-                return to_output_type(res, output_type, order=self.order)
+                return self.transform(X.tocsc())  # TODO keep order
```

Comment:
Not sure what it means about keeping the order? Is it getting transposed here?

Reply:
My bad, it indeed doesn't have a real meaning as `transform` produces a sparse matrix when given a sparse matrix as input. I'll remove the comment.